src/routes/jira/server/jira-server-url-post.ts

```diff
@@ -3,11 +3,64 @@ import axios from "axios";
 import { GitHubServerApp } from "models/github-server-app";
 import { isValidUrl } from "utils/is-valid-url";
 
+interface MessageAndCode {
+	errorCode: string;
+	message: string;
+	statusCode: number;
+}
+
+interface GheServerUrlErrorResponses {
+	[key: string | number]: MessageAndCode;
+}
+
+interface GheServerUrlErrors {
+	codeOrStatus: GheServerUrlErrorResponses
+}
+
+const gheServerUrlErrors: GheServerUrlErrors = {
+	codeOrStatus: {
+		invalidUrl: {
+			errorCode: "GHE_ERROR_1",
```

Comment:
Is there a reason we don't want to use `GHE_ERROR_INVALID_URL`?

Reply:
Happy to discuss alternative options for the error codes. What are you thinking for the other 3?
`GHE_ERROR_ENOTFOUND`
`GHE_ERROR_502` / `GHE_SERVER_BAD_GATEWAY`
`GHE_ERROR_DEFAULT`

?

Reply:
Looks good. `GHE_SERVER_BAD_GATEWAY` is better than `GHE_ERROR_502` in my personal opinions.

Reply:
I agree.